openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -400,22 +402,23 @@ class WaffleFlag(object):
     # use a WeakSet so these instances can be garbage collected if need be
     _class_instances = WeakSet()
 
-    def __init__(self, waffle_namespace, flag_name):
+    def __init__(self, waffle_namespace, flag_name, module_name):
```

Comment:
Because there are many repos that might use these classes, including those outside of the org, I think we should probably make this a kwarg for now.  When we move these to edx-django-utils, we can make it a regular argument.